MapboxNavigation/InstructionsCardContainerView.swift

```diff
@@ -47,7 +47,7 @@ public extension InstructionsCardContainerViewDelegate {
 }
 
 /// :nodoc:
```

Comment:
We’re going to need documentation for this class and its properties as we turn guidance cards into an officially supported feature.